ida-customizations/ida-course/exercise-week4.md

```diff
@@ -0,0 +1,24 @@
+---
+title: "Coding Exercise 3"
+order:
+description: Add a new stored type to the Final Exam
+tags:
+  - cosmos-sdk
+  - ignite-cli
+---
+
+# Coding Exercise 3
+
+**Coding Exercise 3** is now accessible in your personal repo. Click [here](https://git.academy.b9lab.com/ida-p5-final-exam/student-projects) to go to the Academy Gitlab server and open your project's merge requests page.
+
+To complete this exercise, in effect you need to a new stored type with Ignite CLI. An experienced developer can tackle it in 2 hours.
```

Comment:
Missing word: 

_...in effect you need to [SOMETHING] a new stored type..._

Reply:
```suggestion
To complete this exercise, in effect you need to create a new stored type with Ignite CLI. An experienced developer can tackle it in 2 hours.
```